pymodels/SI_V25_04/segmented_models.py

```diff
@@ -113,7 +113,7 @@ def dipole_bc(m_accep_fam_name, simplified=False):
 
 def dipole_b1(m_accep_fam_name, simplified=False):
     """Segmented B1 dipole model."""
-    src_point_angle = 3.2e-3  # [rad] - at SI-01C1:MA-B1 for Carcará.
+    src_point_angle = 3.0e-3  # [rad] - at SI-01C1:MA-B1 for Carcará.
 
     segtypes = {
```

Comment:
@Gabrielrezende-asc , take this opportunity to add a comment saying that the half segmented model for each magnet is downstream from its longitudinal center (i should have done it years ago).